guides/doc-Content_Management_Guide/topics/Using_ISS.adoc

```diff
@@ -26,7 +26,39 @@ You cannot use ISS to synchronize content from {ProjectServer} to {SmartProxySer
 {SmartProxyServer} supports synchronization natively.
 For more information, see {PlanningDocURL}chap-Documentation-Architecture_Guide-Capsule_Server_Overview[{SmartProxyServer} Overview] in _Planning for {ProjectNameX}_.
 
-==== Air gapped
+=== Configuration Definitions
+There are different ways of using ISS to synchronize content between two {ProjectServer}s.
+
+.Connected
+In a connected scenario, two {ProjectServer}s are connected and are able to communicate with each other.
+
+ifndef::satellite[]
+image::Connected.png[title="The {Project} ISS Connected use case", alt="The {Project} ISS Connected use case"]
+endif::[]
+
+.Disconnected
+In a disconnected scenario, there is the following setup:
+* Two {ProjectServer}s are connected to each other.
+* One {ProjectServer} is connected to the Internet while the other {ProjectServer} is not connected to the Internet.
+This is referred to as the disconnected {ProjectServer}.
+* The disconnected {ProjectServer} is completely isolated from all external networks but can communicate with a connected {ProjectServer}.
+
+ifndef::satellite[]
+image::Non-Airgapped-Disconnected.png[title="The {Project} ISS Non Airgapped Disconnected use case", alt="The {Project} ISS Airgapped Disconnected use case"]
+endif::[]
+
+
+.Air-gapped Disconnected
+In an air-gapped scenario, both {ProjectServer}s are completely disconnected from each other.
```

Comment:
My preference is to avoid using the name of the scenario to define the scenario:

```
In this setup, both $servers are isolated from each other by their networks
```